spec/javascripts/app/document-capture/components/suspense-error-boundary-spec.jsx

```diff
@@ -32,17 +31,13 @@ describe('document-capture/components/suspense-error-boundary', () => {
       throw new Error();
     };
 
-    sinon.stub(console, 'error').callsFake(() => {});
-
     const { findByText } = render(
       <SuspenseErrorBoundary fallback="Loading" errorFallback="Error">
         <Child />
       </SuspenseErrorBoundary>,
     );
 
     expect(await findByText('Error')).to.be.ok();
-
-    // eslint-disable-next-line no-console
-    console.error.restore();
+    expect(console).to.have.logged();
```

Comment:
do we want to check for a specific message to have been logged? Or does it not matter

Reply:
I implemented support for it with the expectation that it'd be pretty common (ideal) to verify the specific message:

https://github.com/18F/identity-idp/blob/3a63a7c4a199a6f52e37dec5e199473f8356e970/spec/javascripts/support/console.js#L20-L29

(Aside: Just noticed an unintended `"` there)

The reason I didn't use it here is because the error message is a pretty long stack trace. One option that would probably be helpful anyways is to overload the method to allow `message` to be passed as a regular expression, 

Then we could do:

```suggestion
    expect(console).to.have.logged(/^Failed to load script:/);
```

Reply:
> One option that would probably be helpful anyways is to overload the method to allow `message` to be passed as a regular expression

Updated in 126b020.

Reply:
Big fan! Thanks